std/algorithm/iteration.d

```diff
@@ -343,9 +349,15 @@ private struct _Cache(R, bool bidir)
         source = range;
         if (!range.empty)
         {
-             caches[0] = source.front;
-             static if (bidir)
-                 caches[1] = source.back;
+            caches[0] = source.front;
+            static if (bidir)
+                caches[1] = source.back;
+        }
+        else // needed, because the compiler cannot deduce, that 'caches' is initialized
```

Comment:
Thank you for adding the comment. Please include a link to the issue that this is fixing so someone can look it up if necessary.

Reply:
I added the number of the issue. I hope this is what you meant with "link", because weblinks to bugreports are quite uncommon in Phobos - I couldn't find a single one...